joomla/joomla-cms
# [6.0] Fulfill InstallerScriptInterface with a trait

### Summary of Changes
This implements a trait to be used in combination with the [InstallerScriptInterface](https://github.com/joomla/joomla-cms/blob/5.3-dev/libraries/src/Installer/InstallerScriptInterface.php)


### Testing Instructions
I have no real idea, how to test it. You need an Extension using a install script with the InstallerScriptInterface. Using this trait should resolve the interface and give you some additional options like:

you can set
- a minimum PHP version which is required
- a minimum Joomla! version which is required
- files to be removed
- creating a dashboard menu onthefly

There are also two "custom" methods to extend preflight/postflight and still benefit from their functionality.


### Link to documentations
Please select:

- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/448
